web-services/security/src/main/java/datawave/security/system/ServerSecurityProducer.java

```diff
@@ -31,17 +27,9 @@
 @ApplicationScoped
 @Exclude(ifProjectStage = DatawaveEmbeddedProjectStageHolder.DatawaveEmbedded.class)
 public class ServerSecurityProducer {
-    // Allow injection of JSSESecurityDomain without having to specify the JNDI name at each injection point.
```

Comment:
I had to split some of the beans/resource into a separate class.  Otherwise there ends up being issues creating the JSSESecurityDomain for a remote query logic when loading the QLF because the DatawaveUserService bean (included in this same class) cannot be created at that time.